storefront-reference-architecture-master/cartridges/app_storefront_base/cartridge/templates/default/product/productDetails.isml

```diff
@@ -4,6 +4,7 @@
         var assets = require('*/cartridge/scripts/assets');
         assets.addJs('/js/productDetail.js');
         assets.addCss('/css/product/detail.css');
+        pdict.product.showPromo=true;
```

Comment:
Please what for consistency in the white spaces and the indentations.

Reply:
i made the show promo to be <isset/> and removed the white spaces